### DISCUSSION TITLE
[BASKETBALL] Is there an easy way to pull games played by a player within a weekly matchup?

### DISCUSSION THREAD

**Discussion by gporceng** [cwendt94/espn-api#521] [ESPN API]
Is there an easy way to pull games played by a player within a weekly matchup? Or a way to see total games played by players for a fantasy roster in a given weekly matchup.

I am trying to pull some metrics from my league, and I keep coming back to how important it is to base weekly stat performances off the total games played by players on a fantasy roster... I am having a hard time figuring out a way to pull the total players that had games within a boxscore, and that's not considering the players that had a game but did not play. Was wondering if this groundwork has already been done, or if I am overthinking how to grab this data.

Thanks! Love using this.

**GO HORNETS**

**Comment by cwendt94** [comment]
I haven't found an easy way to get that information. I think you should be able to use `box_scores` functionality and get every scoring_period data and see if that player has play.
